<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrasladoBienesBiblio</class>
 <widget class="QWidget" name="TrasladoBienesBiblio">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Traslado de Bienes - Material Bibliografico</string>
  </property>
  <widget class="QTableWidget" name="tablaBienes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>901</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="botonAgregarBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Agregar Bien</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="nuevoBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nuevo Bien</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Codigo Bien:</string>
   </property>
   <widget class="QLineEdit" name="cuentaNO">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="claseNO">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="correlativoNO">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="especificoNO">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuenta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Clase:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Correlativo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Especifico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inventario:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="anioInventario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motivo del descargo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalIngreso">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>510</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>510</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QToolButton" name="botonEliminarBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>340</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Eliminar Bien</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Observaciones:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="observaciones">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>901</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>148</number>
   </property>
  </widget>
  <widget class="QPushButton" name="finalizarTransaccion">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>931</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Bien:</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Identificacion del Bien</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuenta: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cuentaAsignada">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Especifico: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreEspecifico">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorBien">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="Spanish" country="ElSalvador"/>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="descripcion">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="fechaAquisicion">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adquirido:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Autor(es): </string>
    </property>
    <widget class="QLineEdit" name="nombreAutor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre(s):</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="isbn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Titulo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titulo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>85</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="motivo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>584</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MB-M2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreUnidad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>200 01</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo Traslado:</string>
   </property>
   <widget class="QRadioButton" name="temporal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Temporal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="definitivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Definitivo</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="nombreDpto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Biblioteca Central</string>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unidad que Recibe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Unidad que Entrega:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="unidadRecibe">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="codigoTraslado">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>126</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo Traslado:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections/>
</ui>
